mtx/sidecar/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "dependencies": {
-    "@sap/cds": "^7",
-    "@sap/cds-mtxs": "^1",
+    "@sap/cds": "^8",
+    "@sap/cds-mtxs": "^2",
     "@sap/xssec": "^3",
     "express": "^4",
     "hdb": "^0",
     "passport": "^0",
-    "@sap/hdi-deploy": "4.8.2"
+    "@sap/hdi-deploy": "^5",
+    "@cap-js/hana": "^1"
   },
   "devDependencies": {
     "sqlite3": "^5"
```

Comment:
Let's use @cap-js/sqlite here as well, this is the dependency section you get with a fresh cds add:

```
  "dependencies": {
    "@cap-js/hana": "^1",
    "@sap/cds": "^8",
    "@sap/cds-mtxs": "^2",
    "@sap/xssec": "^4",
    "express": "^4"
  },
  "devDependencies": {
    "@cap-js/sqlite": "^1"
  },
```

Reply:
Let's also update the Node engines version below to `^18 || ^20`